src/foam/u2/FloatView.js

```diff
@@ -51,22 +64,22 @@ foam.CLASS({
       this.SUPER(p);
 
       this.precision = p.precision;
-      this.min       = p.min;
-      this.max       = p.max;
+      this.min = p.min;
```

Comment:
Why messing up alignment?

Reply:
[Horizontal alignment is discouraged in the Google JavaScript Styleguide](https://google.github.io/styleguide/jsguide.html#formatting-horizontal-alignment). Our linter settings also flag it as an error.

Reply:
I've just updated the guide ;-).